notes-service/src/main/java/org/exoplatform/wiki/service/impl/NoteServiceImpl.java

```diff
@@ -328,18 +327,24 @@ public Page updateNote(Page note, PageUpdateType type, Identity userIdentity) th
     if (userIdentity != null && !Utils.canManageNotes(userIdentity.getUserId(), space, existingNote)) {
       throw new IllegalAccessException("User does not have edit the note.");
     }
-    if (PageUpdateType.EDIT_PAGE_CONTENT.equals(type) || PageUpdateType.EDIT_PAGE_CONTENT_AND_TITLE.equals(type)) {
+    if (PageUpdateType.EDIT_PAGE_CONTENT.equals(type) || PageUpdateType.EDIT_PAGE_CONTENT_AND_TITLE.equals(type)
+        || PageUpdateType.EDIT_PAGE_PROPERTIES.equals(type)) {
       note.setUpdatedDate(Calendar.getInstance().getTime());
     }
     note.setContent(note.getContent());
     Page updatedPage = dataStorage.updatePage(note);
-    NotePageProperties notePageProperties = note.getProperties();
-    if (notePageProperties != null) {
-      notePageProperties.setDraft(false);
-      org.exoplatform.social.core.identity.model.Identity identity =
-                                                                   identityManager.getOrCreateIdentity(OrganizationIdentityProvider.NAME,
-                                                                                                       userIdentity.getUserId());
-      saveNoteMetadata(note.getProperties(), note.getLang(), Long.valueOf(identity.getId()));
+    if (userIdentity != null) {
+      try {
+        NotePageProperties properties = note.getProperties();
+        properties.setNoteId(Long.parseLong(updatedPage.getId()));
```

Comment:
do we need to test if (notePageProperties != null) ?

Reply:
Yes, Done